Kernel/Prekernel/boot.S

```diff
@@ -65,10 +65,10 @@ pml4t:
 
 pdpt
 
-    0: boot_pd0 (0-1GB)
-    1: n/a      (1-2GB)
-    2: n/a      (2-3GB)
-    3: n/a      (3-4GB)
+    0: boot_pd0             (0-1GB)
```

Comment:
Please explain in the commit message what you actually try to do here a bit (with the page tables) and why it's needed - this is not understandable just from looking at the assembly code alone.

Reply:
Done